test/test_foreach.py

```diff
@@ -1501,7 +1501,7 @@ def check_autodiff_sample(op, sample, dtype, is_inplace):
         return (
             False,
             "Trying to set a forward gradient that has a different size than that of the original Tensor, "
-            "this is not supported. Tensor is of size [] while the given forward gradient is of size [1, 1].",
+            "this is not supported. Tensor is of size [] while the given forward gradient is of size [1",
```

Comment:
Looks like an accidental change?

Reply:
No this is intentional, cuz now we have 1D shapes instead of 2D, but if we do the standalone test, we'd be able to not taint these

Reply:
undid this change